staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1090,6 +1129,8 @@ func (c *cacheWatcher) process(ctx context.Context, initEvents []*watchCacheEven
 			return
 		case <-c.timer.C:
 			return
+		case <-ticker.C:
+			eventBufferUsage.WithLabelValues(objType, fmt.Sprintf("%v", cap(c.input))).Observe(channelUsage(cap(c.input), len(c.input)))
```

Comment:
This metric doesn't make sense to me - there can be hundreds or thousands of watchers with the same (type, len) and they will all be updating the same thing.
This doesn't provide any value - please remove this metric.

Reply:
How about apiserver_watch_cache_event_blocked_count ? What we want to know is whether there are some watchers that are very slow.

Reply:
If we can compute this metric in a clever way without much overhead, that conceptually sounds reasonable.